tempesta_fw/http_parser.c

```diff
@@ -786,84 +674,112 @@ do {									\
 		 */							\
 		r = TFW_POSTPONE;					\
 		__fsm_const_state = to;					\
+		if (tfw_http_msg_add_data_ptr(msg, &msg->body, data, len)) \
+			return TFW_BLOCK;				\
 		goto done;						\
 	}								\
 	c = *p;								\
 	goto to;							\
 } while (0)
 
-#define __FSM_B_MOVE(to)	__FSM_B_MOVE_n(to, 1)
+#define __FSM_B_MOVE(to)						\
+	__FSM_B_MOVE_n(to, 1)
 
-#define TFW_HTTP_INIT_BODY_PARSING(msg, to_state)			\
+#define TFW_HTTP_INIT_BODY_PARSING(msg)					\
 do {									\
 	TFW_DBG3("parse msg body: flags=%#x content_length=%lu\n",	\
 		 msg->flags, msg->content_length);			\
 	/* RFC 2616 4.4: firstly check chunked transfer encoding. */	\
```

Comment:
Ok, for now, but don't forget in future to update RFC 2616 references to RFC 723[0-5].